sokoban/src/components/Instruction.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import "./Instruction.css";
+
+interface ModalProps {
+  onClose: () => void
+}
+
+const InstructionModal: React.FC<ModalProps> = ({ onClose }) => {
+  return (
+    <div className="modal-overlay">
+      <div className="modal-content">
+        <h2>Instruction</h2>
+        <p>
+          Sokoban (倉庫番, Sōko-ban, lit.'warehouse keeper') is a puzzle
+          video game in which the player pushes boxes around in a warehouse,
```

Comment:
Instruction text needs to be updated (we will look at it together) :)

Reply:
Okay, I can add later :)